src/tempDll/Debug-Net45/netstandard1.1/Microsoft.Azure.Management.RecoveryServices.Backup.deps.json

```diff
@@ -0,0 +1,480 @@
+{
+  "runtimeTarget": {
+    "name": ".NETStandard,Version=v1.1/",
+    "signature": "c965178d804b1a4b428d6d8c61bffeff0a1f2ce9"
+  },
+  "compilationOptions": {},
```

Comment:
For now, we can remove folders 
src\tempDll\Debug-Net45\netstandard1.1
src\tempDll\Debug-Net45\netstandard1.5